js/control/TrackAnalysis.js

```diff
@@ -443,7 +446,7 @@ BR.TrackAnalysis = L.Class.extend({
                     $(
                         `<td class="track-analysis-distance track-analysis-distance-total">${this.formatDistance(
                             totalDistance
-                        )} km</td>`
+                        )} ${i18next.t('footer.kilometer-abbrev')}</td>`
```

Comment:
Thanks for the PR! If you are done, do not forget to:

1. add dynamic i18n keys in `locales/keys.js`
2. run `gulp i18next` to regenerate English translations 
3. I'll push the new content to Transifex then.